docs/source/traits_user_manual/defining.rst

```diff
@@ -312,7 +312,9 @@ the table.
 |                  | None, *kw* = None, *allow_none* = True, *adapt* = None,  |
 |                  | *module* = None, \*\*\ *metadata*] )                     |
 +------------------+----------------------------------------------------------+
-| List             | List([*trait* = None, *value* = None, *minlen* = 0,      |
+| List             | List( [*trait* = None, *value* = None, *minlen* = 0,     |
+|                  | *maxlen* = sys.maxint, *items* = True,                   |
```

Comment:
This should say `sys.maxsize`, not `sys.maxint`. There are other occurrences of `sys.maxint` here; I'll fix those as part of this same PR.

Reply:
Fixed.